docs/source/conf.py

```diff
@@ -15,8 +15,9 @@
 import os
 import shutil
 import sys
+from pathlib import Path
 
-sys.path.insert(0, os.path.abspath("../.."))
+sys.path.insert(0, Path("../..").resolve())
```

Comment:
@yuta0821 I think this does not work as doctest and linkcheck are failing to import ignite:
- https://github.com/pytorch/ignite/runs/5259665461?check_suite_focus=true